src/workshops/node-postgres/index.md

```diff
@@ -76,27 +76,47 @@ This script executes the SQL commands inside `/database/init.sql` against your D
 
 ### Connecting to the database
 
-We can query our DB manually from the terminal using `psql`, but that doesn't help us build an app. We need a way for our Node server to connect to the DB.
+We can query our DB manually from the terminal using `psql`, but that doesn't help us build an app. We need a way for our Node server to connect to the DB. To do this we use the [`node-postgres`](https://node-postgres.com) library. You need to install this into the project with:
+
+```shell
+npm install pg
+```
+
+Our app needs to know the database's address. Postgres runs a local server so you can talk to the DB. The full URL (also known as the "connection string") for your database will be:
 
-To do this we use the [`node-postgres`](https://node-postgres.com) library (called `pg` on npm). It's listed as a dependency in the `package.json` file, so was automatically downloaded when you ran `npm install`.
+```
+postgres://learn_pg_user:@localhost:5432/learn_pg
+```
 
-Take a look at `/database/connection.js`. We import the `pg` library, then connect to the right DB by passing in a "connection string" (a URL identifying the DB and user).
+You _could_ hard-code this URL into our app code, but this address is only correct for the database running on your computer. When someone else clones your repo they'll have their own DB set up.
 
-{% disclosure "More info about the connection string" %}
+### Environment variables
 
-The connection string is a URL pointing to your exact Postgres DB, including the user and password. It is structured like this:
+It's best to read configuration like this from an "environment variable" (env vars). This is a value set in the shell before you start your application. For example it's common to read the `PORT` env var to know what port your app should listen on.
```

Comment:
"This is a value that is made available to the terminal when you start your application"?

"set in the shell" doesn't feel explicit enough😅

Reply:
Yeah there is no accurate way to describe this without the context of how programs actually run :/ I think "made available to the terminal" might be a bit misleading, since the terminal application isn't really involved.

Maybe an example would make it clearer?

> This is a value you can set in your shell before you start your application. E.g. `PORT=3000 node server.js`. You server can then read this value to know what port your app should listen on.

Reply:
yep I like the example! also adds some more clarity to the need for dotenv over doing it manually